src/ModbusAnalyzer.cpp

```diff
@@ -288,11 +142,13 @@ void ModbusAnalyzer::WorkerThread()
 
             // Now we'll process the rest of the data based on whether the transmission is coming from the client or a server device
             if( mSettings->mModbusMode == ModbusAnalyzerEnums::ModbusRTUClient ||
-                mSettings->mModbusMode == ModbusAnalyzerEnums::ModbusASCIIClient )
+                mSettings->mModbusMode == ModbusAnalyzerEnums::ModbusASCIIClient ||
+                ((mSettings->mModbusMode == ModbusAnalyzerEnums::ModbusRTUBoth ||
+                mSettings->mModbusMode == ModbusAnalyzerEnums::ModbusASCIIBoth) && !processingResponse) )
```

Comment:
Note to self: Is there any concern that the new "Both" mode might not work properly if the first message decided is a response, because the request occurred before recording started?

Reply:
An easy way to test this in Logic 2 is to take a capture, and trim the capture such that the data begins with a response.

Reply:
Ah, I see you already mentioned this exact problem in your comment with screenshots.